HLTrigger/Configuration/python/HLT_75e33/modules/ticlTrackstersCLUE3DHigh_cfi.py

```diff
@@ -39,30 +39,75 @@
         type = cms.string('CA')
     ),
     pluginPatternRecognitionByCLUE3D = cms.PSet(
-        algo_verbosity = cms.int32(0),
-        criticalDensity = cms.double(0.6),
-        criticalEtaPhiDistance = cms.double(0.025),
-        criticalSelfDensity = cms.double(0.15),
-        criticalXYDistance = cms.double(1.8),
-        criticalZDistanceLyr = cms.int32(5),
-        densityEtaPhiDistanceSqr = cms.double(0.0008),
-        densityOnSameLayer = cms.bool(False),
-        densitySiblingLayers = cms.int32(3),
-        densityXYDistanceSqr = cms.double(3.24),
-        eid_input_name = cms.string('input'),
-        eid_min_cluster_energy = cms.double(1),
-        eid_n_clusters = cms.int32(10),
-        eid_n_layers = cms.int32(50),
-        eid_output_name_energy = cms.string('output/regressed_energy'),
-        eid_output_name_id = cms.string('output/id_probabilities'),
-        kernelDensityFactor = cms.double(0.2),
-        minNumLayerCluster = cms.int32(2),
-        nearestHigherOnSameLayer = cms.bool(False),
-        outlierMultiplier = cms.double(2),
-        rescaleDensityByZ = cms.bool(False),
-        type = cms.string('CLUE3D'),
-        useAbsoluteProjectiveScale = cms.bool(True),
-        useClusterDimensionXY = cms.bool(False)
+    algo_verbosity = cms.int32(0),
+    criticalDensity = cms.vdouble(
+      4,
```

Comment:
Do you have specific reasons and validation plots to justify moving from `0.6` to `4`?